packages/web/src/common/store/profile/sagas.js

```diff
@@ -456,26 +456,29 @@ function* watchFetchFollowUsers(action) {
 }
 
 function* fetchFolloweeFollows(action) {
-  const { handle } = action
+  const { handle, limit, offset } = action
   if (!handle) return
-  const audiusBackendInstance = yield getContext('audiusBackendInstance')
+  const audiusSdk = yield getContext('audiusSdk')
+  const sdk = yield call(audiusSdk)
   const profileUserId = yield select((state) => getProfileUserId(state, handle))
+  const currentUserId = yield select(getUserId)
   if (!profileUserId) return
-  const followeeFollows = yield call(
-    audiusBackendInstance.getFolloweeFollows,
-    profileUserId,
-    action.limit,
-    action.offset
-  )
+  const response = yield call([sdk.users, sdk.users.getMutualFollowers], {
+    userId: Id.parse(currentUserId),
+    id: Id.parse(profileUserId),
+    limit,
+    offset
+  })
+  if (!response.data) return
 
-  const followerIds = yield call(cacheUsers, followeeFollows)
+  const followerIds = yield call(cacheUsers, response.data)
```

Comment:
These probably need to be run through a SDK adapter before they get put into the cache.